.github/workflows/ci.yml

```diff
@@ -80,10 +80,10 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install npm dependencies
-        run: npm run ci --if-present
+        run: npm run install --if-present
 
       - name: Build
-        run: npm run build --if-present
+        run: npm run build --if-present --env.NODE_ENV=development
 
       - name: Test
         run: npm run test --if-present
```

Comment:
Either remove this step, or remove the default `npm test` command from `package.json` since it will cause the build to fail

Reply:
Ok, its working now